### PR TITLE
Add semi arc sign simple & ellipse markers, add missing arrow sign to ellispe marker

### PR DESCRIPTION
**Pull request by nirvn**
## Description

This PR adds a few useful signs to simple and ellipse markers.
image

### CODE REVIEW

File: src/core/symbology/qgsmarkersymbollayer.h [hunk 1]
```diff
@@ -74,6 +74,9 @@ class CORE_EXPORT QgsSimpleMarkerSymbolLayerBase : public QgsMarkerSymbolLayer
       Octagon, //!< Octagon (since QGIS 3.18)
       SquareWithCorners, //!< A square with diagonal corners (since QGIS 3.18)
       AsteriskFill, //!< A filled asterisk shape (since QGIS 3.18)
+      SemiArc, //!< A line-only semi arc (since QGIS 3.20)
```
Comment (nyalldawson) [inline src/core/symbology/qgsmarkersymbollayer.h:77]: Should this be "half arc" for clarity?

**Review by nyalldawson — APPROVED** [review]

File: src/core/symbology/qgsmarkersymbollayer.h (hunk 1 above)
Comment (nirvn) [reply src/core/symbology/qgsmarkersymbollayer.h:77]: Yeah maybe -- you want only the description changed or the shape enum / string too?

Comment (nyalldawson) [reply src/core/symbology/qgsmarkersymbollayer.h:77]: Probably best to do it all....

**Comment by nirvn** [comment]
@nyalldawson , done.

**Pull request merged**